db/dataset_eval.py

```diff
@@ -298,6 +298,27 @@ def _create_job(connection, dataset_id, normalize, eval_location, filter_type=No
     job_id = result.fetchone()[0]
     return job_id
 
+def get_pending_jobs_for_user(user_id):
```

Comment:
From the name of this function it's not obvious that you are actually getting just jobs for remote evaluation.